### DISCUSSION TITLE
How specific should my location be?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#27243] [New to GitHub] [answered]
I have been a part of the GitHub community for about a month or so, and I just started contributing to an international project; however, I set my location on my [profile](https://github.com/IRod22) to the country I live in (the United States), but I've seen people have entered more specific locations like Calgary in Canada. So how specific should an account location be?

**Comment by ghost** [comment, marked as answer]
as you want .. , that for you 
i prefer u mention your country only

**Reply by shafiqullahsafi** [reply]
my country Afghanistan

**Comment by shafiqullahsafi** [comment]
My email server show error

**Reply by ghost** [reply]
??? That is not the scope of this thread.

\cc: @github-staff

**Comment by shafiqullahsafi** [comment]
IP address
